streams/uk.m3u

```diff
@@ -9,6 +9,8 @@ https://5f3e23ac71915.streamlock.net:4434/live/livestream1.sdp/playlist.m3u8
 https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/akaal_tv/hls1_smart_akaal/akaal_tv.m3u8
 #EXTINF:-1 tvg-id="AkaalTV.uk" status="online",Akaal TV (360p) [Not 24/7]
 http://akaal.zecast.net/akaal-live/smil:akaaltv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Apnatv.in" status="online",APNA TV (720p)
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="Apnatv.uk" status="online",APNA TV (720p)
```

You need to add a Apnatv.uk entry to https://github.com/iptv-org/database/blob/master/data/channels.csv

Reply:
Can you please add the channel i cant seem to be able to update
channels.csv please add

Reply:
Added in https://github.com/iptv-org/database/pull/274